internal/server/server.go

```diff
@@ -1,14 +1,19 @@
 package server
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"embed"
 	"io/fs"
+	"io/ioutil"
```

Comment:
The stdlib package `io/ioutil` is discouraged for new code from 1.16 onwards., there should be equivalent methods in the `io` or `os` packages.